docs/data-integrations/yugabytedb.mdx

```diff
@@ -13,11 +13,12 @@ This handler is implemented using `psycopg2`, a Python library that allows you t
 
 The required arguments to establish a connection are as follows:
 
-* `user` is the database user.
-* `password` is the database password.
-* `host` is the host name, IP address, or URL.
-* `port` is the port used to make TCP/IP connection.
-* `database` is the database name.
+- `user` is the database user.
+- `password` is the database password.
+- `host` is the host name, IP address, or URL.
+- `port` is the port used to make TCP/IP connection.
+- `database` is the database name.
+- `schema` Schema to which your table is asscociated.
```

Comment:
Please correct this line to the following:

> \- \`schema\` is the schema to which your table belongs.

Reply:
Done